From 6a1c600bcb6288c021eafdef516b8313c60505a3 Mon Sep 17 00:00:00 2001
From: Mikko Gronoff <mikko.gronoff@qt.io>
Date: Fri, 17 Mar 2017 14:42:12 +0200
Subject: [PATCH] qt3d: include qcoreapplication.h header in
 qtquickscene2dplugin.cpp

Fix build break in qt3d:
Add #include <QtCore/qcoreapplication.h> to qtquickscene2dplugin.cpp
to prevent build break due unknown Q_COREAPP_STARTUP_FUNCTION macro.
---
 src/quick3d/imports/scene2d/qtquickscene2dplugin.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/quick3d/imports/scene2d/qtquickscene2dplugin.cpp b/src/quick3d/imports/scene2d/qtquickscene2dplugin.cpp
index c67271a..224b24f 100644
--- a/src/quick3d/imports/scene2d/qtquickscene2dplugin.cpp
+++ b/src/quick3d/imports/scene2d/qtquickscene2dplugin.cpp
@@ -40,6 +40,8 @@
 #include <private/qrenderaspect_p.h>
 #include <Qt3DQuickScene2D/private/qt3dquick3dscene2d_p.h>
 
+#include <QtCore/qcoreapplication.h>
+
 QT_BEGIN_NAMESPACE
 
 static void initScene2dPlugin()
